_source/api/logzio-public-api.yml

```diff
@@ -4813,3 +4932,369 @@ definitions:
           $ref: '#/definitions/ShippingTokensModel'
       pagination:
         $ref: '#/definitions/Pagination'
+
+
+  # ::::: Alert
+  Aggregation:
+    type: object
+    description: Specifies a trigger condition that is the threshold.
+    properties:
+      aggregationType:
+        type: string
+        description: Selects the type of aggregation for the trigger condition.
+        enum:
+          - SUM
+          - MIN
+          - MAX
+          - AVG
+          - COUNT
+          - UNIQUE_COUNT
+          - NONE
+      fieldToAggregateOn:
+        type: string
+        description: Selects the field on which to run the aggregation for the trigger condition.
```

Comment:
We have a bunch of conditions regarding the aggregations and group by (nothing new)
1. fieldToAggregateOn can't be in the group by fields.
2. aggregationType is COUNT -> fieldToAggregateOn must be null, and group by fields must not be empty
3. aggregationType is NONE -> fieldToAggregateOn must be null, and group by fields must be empty (or null)
4. aggregationType is any of the other types -> fieldToAggregateOn must not be null